core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,64 @@ var AdminPage = function(){
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="statusMessage"]'));
 
+  // The reload functions are used for mobile testing
+  // done via Browserstack. These functions may cause
+  // a problem when used to run tests directly on Travis.
+
+  var explorationElements = element.all(by.css(
+    '.protractor-test-reload-exploration-row'
+  ));
+
+  var reloadAllExplorationsButtons = element.all(by.css(
+    '.protractor-test-reload-all-explorations-button'
+  ));
+
+  var reloadCollectionButton = element.all(by.css(
+    '.protractor-test-reload-collection-button'));
```

Comment:
`reloadCollectionButtons` to reflect plurality here.

Reply:
Done, thanks!